drivers/ethernet/eth_adin2111.c

```diff
@@ -837,6 +981,12 @@ static int adin2111_init(const struct device *dev)
 		return ret;
 	}
 
+	/*
+	 * Setting this for all modes, RM rev.0 Table 3, t3 >= 50ms
```

Comment:
```suggestion
	 * Setting this for all modes, RM rev. A Table 3, t3 >= 50ms
```

Reply:
[Publicly available RM](https://www.analog.com/media/en/technical-documentation/data-sheets/adin2111.pdf) is revision A.